Project/Project/Areas/Identity/Data/AuthDbContext.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using Project.Areas.Identity.Data;
+
+namespace AuthSystem.Data;
+
+public class AuthDbContext : IdentityDbContext<ApplicationUser>
+{
+    public AuthDbContext(DbContextOptions<AuthDbContext> options)
+        : base(options)
+    {
+    }
+
+    protected override void OnModelCreating(ModelBuilder builder)
+    {
+        base.OnModelCreating(builder);
+        // Customize the ASP.NET Identity model and override the defaults if needed.
+        // For example, you can rename the ASP.NET Identity table names and more.
+        // Add your customizations after calling base.OnModelCreating(builder);

```

Comment:
Check if the user has all of those attributes
<img width="185" alt="image" src="https://github.com/Maxiboy441/PixelPrime/assets/116082321/94095138-5ca0-4186-b1c6-d454a515ecdd">
if not, add them